implementations/c/include/ockam/log.h

```diff
@@ -88,9 +89,9 @@ typedef enum {
 extern "C" {
```

Comment:
let's confirm that the way we `extern "C"` is consistent across all all our headers